### DISCUSSION TITLE
musings/making-and-losing-friends-in-life/

### DISCUSSION THREAD

**Discussion by giscus** [Jitesh117/blog_comments#9] [Q&A]
# musings/making-and-losing-friends-in-life/

Friends are the one thing in our lives which can change the outlook of anything which is happening in our lives. Having their mere presence can really amplify any happiness.
Growing up as a kid whose father worked in the Indian Air Force, I had to switch cities a lot. While it meant I got to make new friends every 3-5 years, it also meant losing most of my friends whenever I changed places.

https://jitesh117.github.io/musings/making-and-losing-friends-in-life/

**Comment by darkrai343** [comment]
Great article!
